torch/utils/_config_module.py

```diff
@@ -414,7 +438,7 @@ def load_config(self, maybe_pickled_config: Union[bytes, Dict[str, Any]]) -> Non
         else:
             config = maybe_pickled_config
         for k, v in config.items():
-            if k in self._config:
+            if k in self._config and self._config[k].alias is None:
```

Comment:
Should we warn vi a a log line if we skip setting due to alias being set?

Reply:
I actually kind of think we should hard error here? We should never have a transitionary state where a pickled config has BOTH the alias and the destination, it should either only ever be one or the other. Furthermore, if I have a pickled config that has the alias set, I *should* set the destination to it, this makes the alias semantics symmetric with how regular sets would work and makes old pickles compatible with newer versions of PT.